sdk/nodejs/provider.ts

```diff
@@ -2222,7 +2247,14 @@ export namespace yaml {
     export class ConfigFile extends CollectionComponentResource {
         constructor(name: string, config?: ConfigFileOpts, opts?: pulumi.ComponentResourceOptions) {
             super("kubernetes:yaml:ConfigFile", name, config, opts);
-            const text = fs.readFileSync(config && config.file || name).toString();
+            const fileId = config && config.file || name
+            let text = "";
+            if (fileId.startsWith("http://") || fileId.startsWith("https://")) {
```

Comment:
What is the full set of rules that `kubectl` uses for interpreting an argument to `-f`?  Are we doing the same?